filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. |
+-->
+
+
+### Example
+
+FileWriter V2 may be configured with cURL.
+
+Using SIMPLON:
+```sh
+curl \
+    -X PUT \
+    -H "Content-Type: application/json" \
+    -d '{"mode": {"value": "enabled"}, "v2_preview_enabled": {"value": true}}' \
+    $ADDRESS_OF_DCU/filewriter/api/1.8.0/config
+```
+
+## Data Access
+
+The files are stored locally on the DCU and have to be transferred to the user computer, this can also be done via cURL using the SIMPLON API.
+
+```sh
+# list available files
+curl $ADDRESS_OF_DCU/filewriter/api/1.8.0/status/files
+
+# file access
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_master.h5
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_data_<file_number>.h5
+```
+
+For further details on available commands and status parameters please refer to the [SIMPLON API documentation](https://media.dectris.com/210607-DECTRIS-SIMPLON-API-Manual_EIGER2-chip-based_detectros.pdf).
+
+## Multi-channel Data
+
+One of the main new features of FileWriter V2 is the support of multi-channel data, i.e. each image consists of multiple channels, one per enabled threshold/difference mode. Therefore, the dataset `/entry/data/data` is four-dimensional, i.e. `[number_of_images, number_of_channels, width, height]`. The channels are identified via strings given in the [_AXISNAME_ field](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) `channel` of the _NXdata_ class[^2]. For every channel name listed in this field, there exists a subgroup of the [_NXdetector_]() group of type _NXdetector_channel_[^1] with the same name followed by the suffix `_channel`. This group contains channel-specific metadata, for example `threshold_energy`, `flatfield` or `pixel_mask`.
+
+An _NXdetector_channel_ group could contain every field that is allowed for NXdetector and that makes sense to be given per channel.
+
+[^2]: Currently, _AXISNAME_ is defined to be of type [_NX_NUMBER_](https://manual.nexusformat.org/nxdl-types.html#nx-number), but there is an [open PR](https://github.com/nexusformat/NIAC/issues/97) to allow [_NX_CHAR_](https://manual.nexusformat.org/nxdl-types.html#nx-char) as well.
+
+#### Example
+
+For example, if `threshold_1`, `threshold_2` and `difference_mode` are enabled, we would get the following:
+
+```
+data: NXdata
+  @signal = "data"
+  @axes = ["image_number", "channel", ".", "."]
+  @image_number_indices = 0
+  @channel_indices = 1
+  image_number = [1, ..., number_of_images]
```

Comment:
This array is actually `[0 + image_id_offset, ..., number_of_images + image_id_offset]`. Since `image_id_offset` defaults to `1`, what I wrote in the example is the default case.
In simplon v1 it was `image_nr_offset`, iirc this is why we first chose "image_number". I think it's a good idea to change it to "image_id" though, since that will match simplon v2.

Reply:
Regarding the renaming, it won't be part of the current bugfix release, so I would suggest to update the documentation when the change is in a release.

Reply:
The change for fw2 (image_number -> image_id) will be included in the next bugfix release, so I added the changes for the documentation in this [commit](https://github.com/soph-dec/documentation/pull/1/commits/353065b67577247ad220ebdae13780605164d14f).

Regarding the global `image_id_offset` for SIMPLON v2, I will open a ticket in Jira.